llvm/lib/CodeGen/RDFRegisters.cpp

```diff
@@ -126,13 +126,10 @@ std::set<RegisterId> PhysicalRegisterInfo::getAliasSet(RegisterId Reg) const {
 std::set<RegisterId> PhysicalRegisterInfo::getUnits(RegisterRef RR) const {
   std::set<RegisterId> Units;
 
-  if (RR.Reg == 0)
```

Comment:
RegisterId 0 is considered a register and the mask is forced to none, so we can let the `RR.isReg()`, `RR.Mask.none()` handle this.